package.json

```
@@ -16,6 +16,7 @@
     "clean:install": "pnpm clean:node_modules && pnpm install --frozen-lockfile",
     "build": "pnpm clean:bundle && turbo ready && turbo build",
     "build:firefox": "pnpm clean:bundle && turbo ready && cross-env __FIREFOX__=true turbo build",
+    "build:eslint": "tsc --build",
```

```suggestion
    "build:eslint": "tsc -b",
```

To match other modules from `dev` branch :)